### ISSUE TITLE
Expose swagger over SSL

### ISSUE THREAD

**Issue by LTroya** [DarkaOnLine/L5-Swagger#60]
I am getting this error when I push my project to a SSL server:
```
Mixed Content: The page at 'https://mydomain.com/api/documentation#!/Auth_Endpoints/post_api_auth_token'
was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 
'http://mydomain.com/docs/api-docs.json'. This request has been blocked; 
the content must be served over HTTPS.
```
Swagger config:
```
/**
 * @SWG\Swagger(
 *     schemes={"https"},
 *     @SWG\Info(
 *         version="1.0.0",
 *         title="API title",
 *         description="API description",
 *         @SWG\Contact(
 *             email="info@gmail.com"
 *         )
 *     )
 * )
 */
```
In my .env file, I have set this: ```APP_URL=https://server.261itsolutions.com```. But no luck, maybe I am missing something important

**Comment by juanabascal** [comment]
I have come up with this issue too. My Laravel application is behind a load balancer so the load balancer receive an HTTPs request and it forwards the request as an HTTP because its an internal call. The index.blade.php file calls the function `l5_swagger_asset` and in the scenario I am it returns an http url.

I have been able to tweak this by autoloading a custom `l5_swagger_asset` function before the one loaded by the vendor. However, I can't do nothing with the `$urlToDocs` variable that represents the path to the swagger json. 

I need to fix this so I'm going to work on a solution to this issue. If it's of your interest I can do a pull request after I have finished the changes. My approach will be:

1. Creating a new config param which would be: `secure_assets`
2. Change the `l5_swagger_asset` function and the `$urlToDocs` so they will adapt the url dynamically depending on the `secure_assets` param.

Best
